### PR TITLE
Added support for Laravel 10

### PR DESCRIPTION
**Pull request by marcoguido**

### CODE REVIEW

**Comment by marcoguido** [comment]
In one of my work projects I quickly needed this library to support Laravel 10.
I **temporary** re-published your package to Packagist at my name, only to allow installing the updated review: because of that I had to rename the package.
Once you'll review my updates, you'll have to merge into `main` all updates made before commit `826a36f` (thus avoiding package rebranding).

**Comment by devanych** [comment]
> In one of my work projects I quickly needed this library to support Laravel 10. I **temporary** re-published your package to Packagist at my name, only to allow installing the updated review in a private library of mine: because of that I had to rename the package. Once you'll review my updates, you'll have to merge into `main` all updates made before commit `826a36f` (thus avoiding package rebranding).

Hi. This update is also relevant to me. Maintainer, tell me if it is possible to merge this PR. Is the package alive?

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "vicklr/materialized-model",
+    "name": "marcoguido/materialized-model",
```
Comment (MithrandirDK) [inline composer.json:2]:
```suggestion
    "name": "vicklr/materialized-model",
```

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -9,10 +9,12 @@ jobs:
             fail-fast: true
             matrix:
                 os: [ubuntu-latest]
-                php: [8.0]
-                laravel: [7.*, 8.*, 9.*]
+                php: [8.0, 8.1, 8.2]
+                laravel: [7.*, 8.*, 9.*, 10.*]
```
Comment (MithrandirDK) [inline .github/workflows/run-tests.yml:13]:
```suggestion
                php: [8.1, 8.2]
                laravel: [8.*, 9.*, 10.*]
```

File: composer.json [hunk 3]
```diff
@@ -12,13 +12,13 @@
     ],
     "require": {
         "php": "^8.0",
-        "illuminate/support": "^7.0|^8.0|^9.0",
-        "illuminate/database": "^7.0|^8.0|^9.0"
+        "illuminate/support": "^7.0|^8.0|^9.0|^10.0",
+        "illuminate/database": "^7.0|^8.0|^9.0|^10.0"
```
Comment (MithrandirDK) [inline composer.json:16]:
```suggestion
        "illuminate/support": "^8.0|^9.0|^10.0",
        "illuminate/database": "^8.0|^9.0|^10.0"
```

File: composer.json [hunk 4]
```diff
@@ -12,13 +12,13 @@
...
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5|^9.5",
+        "phpunit/phpunit": "^8.5|^9.5|^10.0",
         "mockery/mockery": "^1.4",
-        "orchestra/testbench": "^5.0|^6.0|^7.0"
+        "orchestra/testbench": "^5.0|^6.0|^7.0|^8.0"
```
Comment (MithrandirDK) [inline composer.json:21]:
```suggestion
        "orchestra/testbench": "^6.0|^7.0|^8.0"
```

File: README.md [hunk 5]
```diff
@@ -38,12 +38,10 @@ and big trees that must be queried efficiently (e.g. threaded posts).
 <a name="installation"></a>
 ## Installation
 
-Materialized Model works with Laravel 7 onwards. You can add it to your `composer.json` file
-with:
-
-    "vicklr/materialized-model": "~1.0"
-
-Run `composer install` to install it.
+Materialized Model works with Laravel 7 onwards. You can add it to your project by running:
```
Comment (MithrandirDK) [inline README.md:41]:
```suggestion
Materialized Model works with Laravel 8 onwards. You can add it to your project by running:
```

**Comment by MithrandirDK** [comment]
Thank you for the work and I apologise for the lack of action.

I have missed the PR completely, so thank you for providing it and thank you for reminding.

**Pull request merged**